packages/framework/fluid-static/src/rootDataObject.ts

```diff
@@ -137,8 +141,10 @@ export class RootDataObject extends DataObject<{ InitialState: RootDataObjectPro
 const rootDataStoreId = "rootDOId";
 
 /**
- * Container code that provides a single {@link RootDataObject}.  This data object is
- * dynamically customized (registry and initial objects) based on the schema provided to the container runtime factory.
+ * Container code that provides a single {@link RootDataObject}.
+ *
+ * @remarks This data object is dynamically customized (registry and initial objects) based on the schema provided
```

Comment:
Style: I prefer putting a line between the remarks keyword. I think it should read like a section heading.

Reply:
(Not a blocker for his PR.)

Reply:
I don't feel strongly one way or the other, but I love consistency. I think there is probably a jsdoc rule we can enable for this if we want to enforce that style globally.

Reply:
It doesn't look like it supports this out of the box, but the jsdoc plugin does allow creating custom ast-based rules, so we can always do this.

Reply:
```suggestion
 * @remarks
 * 
 * This data object is dynamically customized (registry and initial objects) based on the schema provided.
```

Reply:
For now, I updated all of the multi-line remarks blocks I edited to use the header-like model.